app/views/schemes/_scheme_list.html.erb

```diff
@@ -0,0 +1,39 @@
+<section class="app-table-group" tabindex="0" aria-labelledby="<%= title.dasherize %>">
+  <%= govuk_table do |table| %>
+    <%= table.caption(classes: %w[govuk-!-font-size-19 govuk-!-font-weight-regular]) do |caption| %>
+      <span class="govuk-!-margin-right-4">
```

Comment:
Not a blocker for this PR but I think this caption logic gets repeated a lot and could perhaps become a component.

Reply:
100 agreed, just wanted to push the behemoth over the line at this point, can make a bit prettier next round

Reply:
https://digital.dclg.gov.uk/jira/browse/CLDC-1316